src/commands/service/create-revision.test.ts

```diff
@@ -0,0 +1,167 @@
+import { write } from "../../config";
+import * as azure from "../../lib/git/azure";
+import * as gitutils from "../../lib/gitutils";
+import { createTempDir } from "../../lib/ioUtil";
+import { IBedrockFile } from "../../types";
+import {
+  getDefaultRings,
+  getSourceBranch,
+  makePullRequest
+} from "./create-revision";
+
+jest
+  .spyOn(gitutils, "getCurrentBranch")
+  .mockReturnValueOnce(Promise.resolve("prod"))
+  .mockReturnValue(Promise.resolve(""));
```

Comment:
why do we mock return value once and then mock return value again?

Reply:
Because it's a different return value and it covers the error case for the test case "No source branch"